.gitattributes

```diff
@@ -13,7 +13,7 @@
 * text=auto
 
 # Force the following filetypes to have unix eols, so Windows does not break them
-*.pickle text eol=lf
+*.pickle binary
```

Comment:
Schema's are binary pickles, do not treat them as text or they will get corrupted.